packages/example-app-base/package.json

```diff
@@ -18,7 +18,7 @@
   "devDependencies": {
     "@types/es6-promise": "0.0.32",
     "@types/highlight.js": "9.12.2",
-    "@types/react": "16.3.13",
+    "@types/react": "16.3.14",
```

Comment:
is this bump to 16.3.14 necessary?

Reply:
Yes it is as it fixes the broken react typings that blocked us in the first place :)